zendesk/zendesk_client_service.go

```diff
@@ -241,7 +241,11 @@ func (c *client) getAll(endpoint string, in interface{})([]Ticket, error) {
 	for dataPerPage.NextPage != prevPage {
 		result = append(result, dataPerPage.Tickets...)
 		prevPage = dataPerPage.NextPage
+		if prevPage == "" {
+			break
+		}
```

Comment:
the PR to fix this function is merged. You can pull the laster version to get the updated function.

Reply:
This commit is before your merge.